psycopg_c/psycopg_c/pq/libpq.pxd

```diff
@@ -294,6 +294,8 @@ cdef extern from *:
 #endif
 
 #if PG_VERSION_NUM < 140000
+int PGRES_PIPELINE_SYNC = 10;
```

Comment:
This becomes a module variable, so I think it should be declared static... But probably a `#define PGRES_PIPELINE_SYNC 10` is even better.